docs/install.md

```diff
@@ -245,19 +242,33 @@ If EaseMesh infrastructure has already been installed, the add-on features could
 emctl install --only-add-on --add-ons=ShadowService
 ```
 
+### Install CoreDNS
```

Comment:
I think it's necessary to remind users that the installation will overlap the system coredns image version and configuration.

Reply:
Updated.